core/lib/dal/migrations/20240409102923_drop_l1_batches_fee_account_address.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE l1_batches DROP COLUMN IF EXISTS fee_account_address;
```

Comment:
There [are a couple of other columns](https://github.com/matter-labs/zksync-era/pull/688#discussion_r1426827324) in the same table that aren't used. Do we want to remove them as well, or should it be tackled separately?

Reply:
I'll do it separately today